client/pdo/client/controller/contract_controller.py

```diff
@@ -184,9 +184,17 @@ def ProcessScript(controller, filename, echo=False, identity=None, private_key_f
         subcontroller.bindings.bind('_script_', os.path.basename(os.path.realpath(filename)))
 
         try :
+            saved_line = ''
             cmdlines = ContractController.ParseScriptFile(filename)
             for cmdline in cmdlines :
-                if subcontroller.onecmd(cmdline) : break
+                if cmdline[-1] == '\\' :
```

Comment:
It would be helpful to add a comment here explaining the purpose of this if-else statement. Doesn't have to be anything too long, just a short line about how this is handling multi-line commands.

Reply:
Done